maven-core/src/main/java/org/apache/maven/internal/aether/DefaultRepositorySystemSessionFactory.java

```diff
@@ -113,6 +113,7 @@ public DefaultRepositorySystemSession newRepositorySession( MavenExecutionReques
         Map<Object, Object> configProps = new LinkedHashMap<>();
         configProps.put( ConfigurationProperties.USER_AGENT, getUserAgent() );
         configProps.put( ConfigurationProperties.INTERACTIVE, request.isInteractiveMode() );
+        configProps.put( "maven.startTime", request.getStartTime() );
```

Comment:
Would it make sense to introduce a constant for this?

Reply:
Very good question, likely. The question is where to put it? because it is an implementation detail.